src/drm.cpp

```diff
@@ -1279,22 +1288,56 @@ void drm_unlock_fbid( struct drm_t *drm, uint32_t fbid )
 }
 
 /* Handle the orientation of the display */
-uint64_t get_drm_effective_orientation()
+	void update_drm_effective_orientation(struct drm_t *drm,  struct connector *conn)
```

Comment:
Style:

- Remove the indentation here
- Replace with double space between the two args with a single space